_layouts/default.html

```diff
@@ -7,173 +7,219 @@
+    <link rel="apple-touch-startup-image"
+        media="(device-width: 768px) and (device-height: 1024px) and (orientation: portrait) and (-webkit-device-pixel-ratio: 1)"
+        href="{{ design_guide_version_url }}/icons/apple-touch-startup-image-768x1004.png">
+    <link rel="apple-touch-startup-image"
+        media="(device-width: 768px) and (device-height: 1024px) and (orientation: landscape) and (-webkit-device-pixel-ratio: 2)"
+        href="{{ design_guide_version_url }}/icons/apple-touch-startup-image-1496x2048.png">
+    <link rel="apple-touch-startup-image"
+        media="(device-width: 768px) and (device-height: 1024px) and (orientation: portrait) and (-webkit-device-pixel-ratio: 2)"
+        href="{{ design_guide_version_url }}/icons/apple-touch-startup-image-1536x2008.png">
+    <link rel="icon" type="image/png" sizes="228x228" href="{{ design_guide_version_url }}/icons/coast-228x228.png">
+    <script src="{{ '/assets/js/mermaid.min.js' | relative_url }}"></script>
+    {% if site.search.enabled == true %}
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+    <script src="{{ "/assets/tipuesearch/tipuesearch_content.js" | relative_url }}"></script>
+    <script src="{{ "/assets/tipuesearch/tipuesearch_set.js" | relative_url }}"></script>
+    <script src="{{ "/assets/tipuesearch/tipuesearch.min.js" | relative_url }}"></script>
+    {% endif %}
+</head>
+
+<body>
+    <div id="designguide">
+        <header class="topbar topbar-md-wide designguide-header">
+            <button type="button" class="topbar-btn"><i class="material-icons topbar-btn-icon">menu</i></button>
+            <a href="{{ site.url }}" class="topbar-logo"><img
+                    src="{{ design_guide_version_url }}/img/swedbankpay-logo.svg" alt="logo"></a>
+            <nav class="topbar-nav">
+                <div class="topbar-link-container">
+                    <button type="button" class="topbar-close"><i class="material-icons">close</i></button>
+
+                    {% assign site_url = "/" | relative_url %}
+                    {% assign page_url = page.url | remove: '.html' %}
+
+                    {% comment %}
+                    TODO: Enumerate pages through site.pages instead of hard coded in _config.yml.
+                    Requires a bit sophisticated logic to differ between topbar and sidebar
+                    pages as well as support for page/menu hierarchies.
+                    {% for page in site.pages %}
+                    {% unless page.dir contains 'assets/' %}
+                    * {{ page.title }}
+                    {% endunless %}
+                    {% endfor %}
+                    {% endcomment %}
+
+                    <a {% if page_url == site_url %} aria-current="page" class="active" {% endif %}
+                        href="{{ site_url }}"><span>Home</span></a>
+
+                    {% for nav in site.topbar.navigation %}
```

Comment:
Could the topbar navigation be made dynamic as well? 🤔

Reply:
Probably? But it might be a can of worms.
How can we dynamically determine what to be in the top bar?

Reply:
Good question. Let's not spend any time finding an answer now. 😄